src/main/java/com/gmail/interceptors/UserInterceptor.java

```diff
@@ -0,0 +1,31 @@
+package com.gmail.interceptors;
+
+import lombok.extern.log4j.Log4j;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Log4j
+@Component
+public class UserInterceptor extends HandlerInterceptorAdapter {
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        log.info("Interceptor preHandle");
+        return super.preHandle(request, response, handler);
+    }
+
+    @Override
+    public void postHandle(HttpServletRequest request, HttpServletResponse response, Object handler, ModelAndView modelAndView) throws Exception {
```

Comment:
Line too long please break it into two or more lines 
I could be something like this.
```
public void postHandle(HttpServletRequest request, 
                                      HttpServletResponse response, 
                                      Object handler, 
                                   ModelAndView modelAndView) throws Exception {
```